helm/nginx-ingress-controller-app/templates/controller-service-internal.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: v1
 kind: Service
```

Comment:
Should we avoid to deploy this service if `internal` is disabled? (I meant add an if with the condition)

Reply:
@pipo02mix thanks for reviewing, much appreciated!

There is if condition surrounding whole template
```
{{- if .Values.controller.service.internal.enabled }}
...
```

What am I missing?

Reply:
ahh it was already there. Sorry it was hide in github UI